tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/TypeSuite.scala

```diff
@@ -317,13 +317,13 @@ class TypeSuite extends BaseDottySuite {
 
   test("F[_]") {
     assertTpe("F[_]") {
-      Apply(TypeName("F"), AnonymousParam(None) :: Nil)
+      AnonymousLambda(Apply(TypeName("F"), List(AnonymousParam(None))))
```

Comment:
Will this not be a Lambda only after we disable the possibility to use `_` as placeholder?

Reply:
in scala 3.2 (and scala3 points to 3.2), `_` *is* a placeholder (aka, anonymous parameter for type lambda) but not a _wildcard_.